v22.1/vault-db-secrets-tutorial.md

```diff
@@ -0,0 +1,440 @@
+    token_duration       1h
+    token_renewable      true
+    token_policies       ["default" "roach-client"]
+    identity_policies    []
+    policies             ["default" "roach-client"]
+    ~~~
+
+    You can either copy the `token` from the output of the previous command, or capture a token with the following command (which requires the shell utility [jq](https://stedolan.github.io/jq/)).
+
+    ~~~shell
+     VAULT_CLIENT_TOKEN=`vault token create -policy=roach-client -format=json | jq .auth.client_token | tr -d '"'`
+    ~~~
+
+## Step 4: Connect to CockroachDB with Vault-provisioned credentials
+
+In this phase of the tutorial, we will use credentials provisioned by Vault to access our CockroachDB cluster, emulating the flow that an application dev ops engineer or application service account might use to achieve database access. Therefore, unlike in the first phase, we will not use Vault admin credentials or CockroachDB credentials aquired other than through Vault, since these should not be required.
+
+1. Authenticate to Vault as the CockroachDB user.
+
+    Use the client token to authenticate to Vault with limited permissions.
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    vault login $VAULT_CLIENT_TOKEN
+    ~~~
+    ~~~txt
+    Success! You are now authenticated. The token information displayed below
+    is already stored in the token helper. You do NOT need to run "vault login"
+    again. Future Vault requests will automatically use this token.
+    ~~~
+
+1. Confirm the limited permissions of the role.
+
+    To confirm that you have assumed a role with limited permissions, try listing the current leases on the `crdb-role`, as we did [previously](#). You should see a permissions error.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    vault list sys/leases/lookup/database/creds/crdb-role/
+    ~~~
+    ~~~txt
+    Error listing sys/leases/lookup/database/creds/crdb-role: Error making API request.
+
+    Namespace: admin/
+    URL: GET https://roach-test-vault.vault.bfb2290a-670b-4a10-bedf-5aab18e84d69.aws.hashicorp.cloud:8200/v1/sys/leases/lookup/database/creds/crdb-role?list=true
+    Code: 403. Errors:
+
+    * 1 error occurred:
+      * permission denied
+    ~~~
+
+1. Pull the CockroachDB credentials.
+
+    The only thing this policy does have permission to do is pull credentials for the CockroachDB cluster. Let's do that.
+
+    {% include_cached copy-clipboard.html %}
+    ~~~shell
+    vault read database/creds/crdb-role
+    ~~~
+    ~~~txt
+    Key                Value
+    ---                -----
+    lease_id           database/creds/crdb-role/V3T4UVxeQ9RYsJAk3jZF1Dhl.iMXtW
+    lease_duration     1h
+    lease_renewable    true
+    password           FlOo0p7jMTXjT27hlZZ-H
+    username           v-token-crdb-rol-thfLPlFwex0k9Op0P8qA-1653528652
+    ~~~
+  
+1. Connect to the CockroachDB cluster with your Vault-generated dynamic secret credentials
```

Comment:
Nit: missing a period at the end of this sentence "... your Vault-generated dynamic secret credentials."